pkg/pki/certmanagerpki/certmanager.go

```diff
@@ -22,6 +22,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const uriSanTemplate = "spiffe://%s/%s"
```

Comment:
I'd simply call this const as `spiffeIdTemplate` (this is a SPIFFE ID according to https://spiffe.io/docs/latest/spiffe/concepts/#spiffe-id)